sources/packages/backend/apps/api/src/route-controllers/models/pagination.dto.ts

```diff
@@ -137,7 +137,24 @@ export class ProgramsPaginationOptionsAPIInDTO extends PaginationOptionsAPIInDTO
 
 export class OfferingsPaginationOptionsAPIInDTO extends PaginationOptionsAPIInDTO {
   @IsOptional()
-  @IsIn(["name"])
+  @IsEnum(OfferingIntensity)
+  intensityFilter?: OfferingIntensity;
+
+  @IsOptional()
+  studyStartDateFromFilter?: Date;
+
+  @IsOptional()
+  studyStartDateToFilter?: Date;
```

Comment:
As mentioned by the co-pilot, both studyStartDateFromFilter and studyStartDateToFilter must be validated with class validators. 
 For studyStartDateToFilter `@IsDateString` and `@AllowIf`(custom) can be used. and for `studyStartDateToFilter` `@IsDateString` `IsDateAfter` and `AllowIf` can be used.